bindinfo/session_handle_serial_test.go

```diff
@@ -0,0 +1,540 @@
+	require.True(t, tk.Session().Auth(&auth.UserIdentity{Username: "root", Hostname: "%"}, nil, nil))
+	rows := tk.MustQuery("show global bindings").Rows()
+	require.Len(t, rows, 0)
+	// Simulate existing bindings in the mysql.bind_info.
+	tk.MustExec("insert into mysql.bind_info values('select * from `spm` . `t`', 'select * from `spm` . `t` USE INDEX (`a`)', 'SPM', 'using', '2000-01-01 09:00:00', '2000-01-01 09:00:00', '', '','" +
+		bindinfo.Manual + "')")
+	tk.MustExec("insert into mysql.bind_info values('select * from `spm` . `t0`', 'select * from `spm` . `t0` USE INDEX (`a`)', 'SPM', 'using', '2000-01-02 09:00:00', '2000-01-02 09:00:00', '', '','" +
+		bindinfo.Manual + "')")
+	tk.MustExec("insert into mysql.bind_info values('select * from `spm` . `t`', 'select /*+ use_index(`t` `a`)*/ * from `spm` . `t`', 'SPM', 'using', '2000-01-03 09:00:00', '2000-01-03 09:00:00', '', '','" +
+		bindinfo.Manual + "')")
+	tk.MustExec("insert into mysql.bind_info values('select * from `spm` . `t0`', 'select /*+ use_index(`t0` `a`)*/ * from `spm` . `t0`', 'SPM', 'using', '2000-01-04 09:00:00', '2000-01-04 09:00:00', '', '','" +
+		bindinfo.Manual + "')")
+	tk.MustExec("admin reload bindings")
+	rows = tk.MustQuery("show global bindings").Rows()
+	require.Len(t, rows, 4)
+	require.Equal(t, "select * from `spm` . `t0`", rows[0][0])
+	require.Equal(t, "2000-01-04 09:00:00.000", rows[0][5])
+	require.Equal(t, "select * from `spm` . `t0`", rows[1][0])
+	require.Equal(t, "2000-01-02 09:00:00.000", rows[1][5])
+	require.Equal(t, "select * from `spm` . `t`", rows[2][0])
+	require.Equal(t, "2000-01-03 09:00:00.000", rows[2][5])
+	require.Equal(t, "select * from `spm` . `t`", rows[3][0])
+	require.Equal(t, "2000-01-01 09:00:00.000", rows[3][5])
+
+	rows = tk.MustQuery("show session bindings").Rows()
+	require.Len(t, rows, 0)
+	tk.MustExec("create session binding for select a from t using select a from t")
+	tk.MustExec("create session binding for select a from t0 using select a from t0")
+	tk.MustExec("create session binding for select b from t using select b from t")
+	tk.MustExec("create session binding for select b from t0 using select b from t0")
+	rows = tk.MustQuery("show session bindings").Rows()
+	require.Len(t, rows, 4)
+	require.Equal(t, "select `b` from `spm` . `t0`", rows[0][0])
+	require.Equal(t, "select `b` from `spm` . `t`", rows[1][0])
+	require.Equal(t, "select `a` from `spm` . `t0`", rows[2][0])
+	require.Equal(t, "select `a` from `spm` . `t`", rows[3][0])
+}
+
+func TestDuplicateBindings(t *testing.T) {
+	store, clean := testkit.CreateMockStore(t)
+	defer clean()
+
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t(a int, b int, index idx(a))")
+	tk.MustExec("create global binding for select * from t using select * from t use index(idx);")
+	rows := tk.MustQuery("show global bindings").Rows()
+	require.Len(t, rows, 1)
+	createTime := rows[0][4]
+	time.Sleep(1000000)
```

Comment:
I know this is copied from the original, but can se change it to a const for readability:
```suggestion
	time.Sleep(time.Millisecond)
```